frontend/src/helpers/index.ts

```diff
@@ -20,3 +20,67 @@ export function isElectron(): boolean {
 
   return false;
 }
+
+export function getFilterValueByNameFromURL(key: string, location: any): string[] {
+  const searchParams = new URLSearchParams(location.search);
+  const filtersInURLString = searchParams.get('filters');
+  if (!!filtersInURLString) {
```

Comment:
Invert the logic with an early `[]` return to make the code easier to read.

Reply:
So @joaquimrocha  here we want this to be like this only cause the return [] also acts as a default return when the filters are there in url but the property that you are asking for was not present

Reply:
Maybe I couldn't express what I meant:
You have a big block in `if (!!filtersInURLString) { …`. You could invert the logic and early return:
```javascript
  if (!filterInURLString) {
    return [];
  }

  const …
```

Anyway it's a nitpick.